pkg/azurefile/utils.go

```diff
@@ -265,3 +275,47 @@ func replaceWithMap(str string, m map[string]string) string {
 	}
 	return str
 }
+
+func getAzcopyJob(dstFileshare string) (AzcopyJobState, string, error) {
+	cmdStr := fmt.Sprintf("azcopy jobs list | grep %s -B 3", dstFileshare)
+	out, err := exec.Command("sh", "-c", cmdStr).CombinedOutput()
+	if err != nil && err.Error() != "exit status 1" {
```

Comment:
why it's not `if err != nil && err.Error() != "exit status 1"`? as long as `err != nil`, it should return error? any consideration here?

Reply:
done